app/code/Magebit/Faq/Api/QuestionManagementInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @copyright Copyright (c) 2024 Magebit, Ltd. (https://magebit.com/)
+ * @author    Magebit <info@magebit.com>
+ * @license   MIT
+ */
+
+declare(strict_types=1);
+
+namespace Magebit\Faq\Api;
+
+use Magento\Framework\Exception\CouldNotSaveException;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+interface QuestionManagementInterface
+{
+    /**
+     * Enable a question by its ID.
+     *
+     * @param int $id
+     * @return bool
+     * @throws NoSuchEntityException
```

Comment:
better handle the errors with try and catch. if handled correctly, no need to add throws